File: eval/fixtures/legacy-directory-backed-spring-commerce/src/main/java/com/acme/legacy/orders/OrderFulfillmentController.java
package com.acme.legacy.orders;

@RestController
@RequestMapping("/api/v5/orders")
public class OrderFulfillmentController {
  private final OrderFulfillmentService orderFulfillmentService = new OrderFulfillmentService();

  @PatchMapping("/fulfillment/{orderId}/reprice")
  public ResponseEntity<?> repriceFulfillment(String orderId) {
    return ResponseEntity.ok(orderFulfillmentService.repriceOrder(orderId));
  }
}

File: eval/fixtures/legacy-directory-backed-aspnet-commerce/src/Legacy/Orders/OrderFulfillmentController.cs
namespace Acme.Legacy.Orders;

[ApiController]
[Route("api/v6/orders")]
public class OrderFulfillmentController : ControllerBase
{
  private readonly OrderFulfillmentService _orderFulfillmentService = new OrderFulfillmentService();

  [HttpPatch("fulfillment/{orderId}/reprice")]
  public IActionResult RepriceFulfillment(string orderId)
  {
    return Ok(_orderFulfillmentService.RepriceOrder(orderId));
  }
}
